example/ios/LeveldbExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```

Comment:
what is this file? If it's specific to your ID or setup, could we remove it?

Reply:
Hi. Seems to be a file generated by all Xcode with versions > 9.3 (which was a while ago; right now is version 13+).
https://stackoverflow.com/a/50368149/4469172

Reply:
But I think removing xcshareddata should be good.

Reply:
Ah. according to here xcshareddata should be committed though
https://stackoverflow.com/a/53039267/4469172
`Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace. (37293167)`

So... I guess we should keep them.